deployments/charts/kuma/values.yaml

```diff
@@ -16,6 +16,8 @@ controlPlane:
 
   kdsGlobalAddress: ""
 
+  replicas: 1
```

Comment:
I know why it is on by default, but in reality - do we advise people to run a single copy of the CP?

Reply:
On production? No, you should run at least 2 for HA (and probably with a PodAffinity settings that two of them are not spawned on the same Node). Ideally, this will be controlled by Horizontal Pod Autoscaler, hence the issue I created.

So what's your suggestion for the default value?

Reply:
OK lets go for the HPA apraoch and leave this one as `1`.